Helper/Data.php

```diff
@@ -1785,4 +1786,20 @@ public function getPspReferenceSearchUrl($pspReference, $liveEnvironment)
             $pspReference
         );
     }
+
+    /**
+     * Maps zh_Hans_* locale code to zh_CN
+     * @param $localeCode
+     * @return mixed|string
+     */
+    public function mapLocaleCode($localeCode)
```

Comment:
I think I would prefer to move this to a new locale helper, where the mappings can be added as part of an array constant, what do you think?

Reply:
I think it makes sense since we try move from a generic helper class to multiple more specified ones.
Also the const array could make it more clean

Reply:
@acampos1916 @cyattilakiss Could you check again?